code/src/cutscenes.c

```diff
@@ -235,13 +235,19 @@ void Custcene_OverrideSpiritTemple(void) {
     gGlobalContext->sceneLoadFlag = 0x14;
     gGlobalContext->fadeOutTransition = 0x3;
     gSaveContext.nextCutsceneIndex = 0x0;
-    ItemOverride_PushDungeonReward(DUNGEON_SPIRIT_TEMPLE);
+    if (EventCheck(0x47) == 0) {
```

Comment:
Can you change the indents here to be 4 spaces for consistency with the rest of the file?